src/NosCore.GameObject/Map/MapInstancePortalHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using NosCore.Shared.Map;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NosCore.GameObject.Map
+{
+    public class MapInstancePortalHandler
+    {
+        #region Properties
+
+        public static int SourceX { get; private set; }
+
+        #endregion
+
+        #region Methods
+
+        public static List<Portal> GenerateMinilandEntryPortals(short entryMap, Guid exitMapinstanceId)
```

Comment:
dito and even if we needed it it shouldnt be part of the handler which  should be named controller now